src/averageAges.js

```diff
@@ -20,6 +20,11 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const mens = people.filter(({ sex, died }) => sex === 'm'
+    && (century ? century === Math.ceil(died / 100) : true));
+
```

Comment:
man - its 1 , men - its many , mens doesnt corectly, and you should write ternary operation like this 
`century `
`? century === Math.ceil(died / 100)`
`: true)`

Reply:
Agree, let's format code